### ISSUE TITLE
PDF or all-in-one-page tutorial not available on the site

### ISSUE THREAD

**Issue by sylvaticus** [sympy/sympy#9373]
Hello,
    I may be old fashion but I prefer to read (and make notes) on a pen than read on video.
Currently the tutorial is proposed in one-page-per-section, that makes it really uncomfortable.

I suggest to have an option to have it either available as all-in-one-page (so that users can just print it) or having it available as PDF.

(I am aware of issue #6051 , but that issue refers on build problems in LaTeX while I refer to the availability of precompiled PDF for the end user)

**Comment by debugger22** [comment]
I built this pdf around 10 days back. https://www.dropbox.com/s/rsujktfogzdn23x/sympy-0.7.6-git.pdf?dl=1

Hope this helps!

**Comment by debugger22** [comment]
You can try creating this by yourself. You will have to install all the dependencies before doing that.

```
$ sudo apt-get install graphviz inkscape texlive texlive-xetex texlive-fonts-recommended texlive-latex-extra lmodern librsvg2-bin imagemagick docbook2x;
$ cd doc
$ make pdf  # Follow instructions printed on the screen
```

**Comment by sylvaticus** [comment]
Thank you, it has a very nice look. However I still think that for the end user would be nice to have this pdf (or maybe one without the reference) automatically built and available on the site... after all also the python tutorial is available as pdf..

**Comment by debugger22** [comment]
Perhaps you are right.
@asmeurer @certik should we put pdfs as well on the website?

**Comment by toolforger** [comment]
> should we put pdfs as well on the website?

+1
It's pretty standard to have this actually.

**Comment by moorepants** [comment]
Yes, this can and should be added to the website. Someone just needs to do it manually or add it to the release process.

**Comment by asmeurer** [comment]
We already build pdf versions of the docs (which includes the tutorial) with each release. See https://github.com/sympy/sympy/releases.

**Comment by oscarbenjamin** [comment]
I haven't seen that before but yes there is a 2000 page pdf for the docs.

@sylvaticus is that more or less what you wanted?

**Comment by sylvaticus** [comment]
I still think it is a bit hidden from the general user that don't know much about github releases... it is true that on top of the overall documentation there is a link ("A PDF version of these docs can be found here.") but most new people arrive to the tutorial following the link on the homepage, bypassing that page.

Maybe repeating the link in the "About this tutorial" ? The idea is that the tutorial shoudn't assume that one has already read the documentation first ;-)
